src/main/resources/db/changelog/changes01/21072021_add_column_dis_version.yml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: 21072021_add_column_dis_version
+      author: tesliuk.ua@gmail.com
+      preConditions:
+        - onFail: MARK_RAN
+          not:
+            - columnExists:
+                columnName: dis_version
+                tableName: discount
+                schemaName:  pulic
+      changes:
+        - addColumn:
+            tableName: discount
+            columns:
+              - column:
+                  name: dis_version
+                  type: BIGINT
```

Comment:
let's set up the default version number as 1

Reply:
fixed